### ISSUE TITLE
Dataset split

### ISSUE THREAD

**Issue by wyd2** [zyj15416/TPDM-CBCT2CT#2]
Why is the validation set and test set in this article using the top 10% of the training set? Is there any particular reason for this, and I wonder if the authors can give an explanation?
